src/renderers/WebGLRenderTarget.js

```diff
@@ -12,25 +12,16 @@ THREE.WebGLRenderTarget = function ( width, height, options ) {
 
 	options = options || {};
 
-	this.wrapS = options.wrapS !== undefined ? options.wrapS : THREE.ClampToEdgeWrapping;
-	this.wrapT = options.wrapT !== undefined ? options.wrapT : THREE.ClampToEdgeWrapping;
-
-	this.magFilter = options.magFilter !== undefined ? options.magFilter : THREE.LinearFilter;
-	this.minFilter = options.minFilter !== undefined ? options.minFilter : THREE.LinearMipMapLinearFilter;
-
-	this.anisotropy = options.anisotropy !== undefined ? options.anisotropy : 1;
-
-	this.offset = new THREE.Vector2( 0, 0 );
-	this.repeat = new THREE.Vector2( 1, 1 );
-
-	this.format = options.format !== undefined ? options.format : THREE.RGBAFormat;
-	this.type = options.type !== undefined ? options.type : THREE.UnsignedByteType;
+	this.texture = new THREE.Texture(undefined, undefined, 
+																	 options.wrapS, options.wrapT,
+																	 options.magFilter, 
+																	 options.minFilter !== undefined ? options.minFilter : THREE.LinearFilter,
```

Comment:
@WestLangley Could you clarify the indentation rules? The style just says "Indentation should be tabs", but doesn't specify how to deal with alignment. Are we using tabs for alignment or spaces? If tabs, what is the tab size specification? I use a tabsize of 2; the 18 tabs is necessary to align the subsequent lines with the parenthesis after "new Texture".
I assumed we used tabs by looking e.g. here: https://github.com/mrdoob/three.js/blob/master/src/extras/core/Path.js#L136